src/api/app/models/branch_package.rb

```diff
@@ -185,7 +183,11 @@ def create_branch_packages(tprj)
            p[:package].project != p[:link_target_project]
           opts[:extendvrev] = '1'
         end
-        tpkg.branch_from(oproject, opackage, opts)
+        unless oproject.try(:scmsync).present? || opackage.try(:scmsync).present?
+          opackage = p[:package].name if p[:package].is_a?(Package)
+          oproject = p[:link_target_project].name if p[:link_target_project].is_a?(Project)
```

Comment:
Why move this here? `opackage` and `oproject` are used further down too...

Reply:
changed